src/main/java/gov/nasa/pds/tools/util/PDFUtil.java

```diff
@@ -99,11 +99,11 @@ private synchronized void writeErrorToFile(URI uri, ValidationResult result, Str
     }
   }
 
-  private boolean validatePDF(URI uri, String pdfRef) {
+  private boolean validatePDF(URI uri, String pdfRef) throws IOException {
     boolean pdfValidateFlag = false;
 
-    // Create a parser and auto-detect flavour
     try {
+      // Create a parser and auto-detect flavour
```

Comment:
🇬🇧

Reply:
@nutjob4life 😆 yeah. That is how they spell in the verapdf library so we follow them!